synapse/lib/stormtypes.py

```diff
@@ -7537,6 +7537,14 @@ class View(Prim):
                   'args': (),
                   'returns': {'name': 'Yields', 'type': 'dict',
                               'desc': 'Yields previously successful merges into the view.'}}},
+        {'name': 'updateMergeVoteComment', 'desc': 'Update the comment associated with your vote on a merge request.',
```

Comment:
Lets make these APIs ( and subsequent python APIs and nexus events ) use `set` rather than `update`